### PR TITLE
Add snippet identifier and remove extra spaces

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/databases/database-identifiers.md [hunk 1]
```diff
@@ -23,100 +23,101 @@ ms.author: "sstein"
...
+```sql
+SELECT *
+FROM TableX
+WHERE KeyCol = 124
+```
+
+ Delimited identifiers
+ Are enclosed in double quotation marks (") or brackets ([ ]). Identifiers that comply with the rules for the format of identifiers might not be delimited. For example:
+
+```
```
Comment (CarlRabeler) [inline docs/relational-databases/databases/database-identifiers.md]: add SQL here too

File: docs/t-sql/statements/collations.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Defines a collation of a database or table column, or a collation cast operation
 
 ## Syntax
 
-```
+```sql
```
Comment (CarlRabeler) [inline docs/t-sql/statements/collations.md]: remove SQL here - not used for syntax - only for actual code

**Comment by PRMerger16** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Youssef1313** [comment]
@CarlRabeler , I've addressed your review.

**Pull request merged**